tests/unit/lib/SGVector_unittest.cc

```diff
@@ -194,14 +171,6 @@ TEST(SGVectorTest,complex128_tests)
 	a.set_const(complex128_t(5.0, 6.0));
 	SGVector<complex128_t> b=a.clone();
 
-	// test ::operator+ and []
-	a=a+b;
-	for (index_t i=0; i<a.vlen; ++i)
-	{
-		EXPECT_NEAR(a[i].real(), 10.0, 1E-14);
-		EXPECT_NEAR(a[i].imag(), 12.0, 1E-14);
-	}
-
```

Comment:
you need to replace the call here right? `a=a+b` should be `a=linalg::add(a,b)`, otherwise you won't be getting the same results below...

Reply:
Since this is a test for the + operator, isn't it better to just remove the code?

Reply:
I think this might be the only place left where we test complex numbers though... But yes ideally remove this and add complex128_t add test in linalg unittests

Reply:
Which file should this test for complex addition be added though?
I see a test for the add function in the file : `tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc`
Should the complex test be added here?

Reply:
see my comment [here](https://github.com/shogun-toolbox/shogun/pull/4958/files/daf8ae841f6bda16a2dd0732fa05398f3cab59ba#diff-c6e86f9deed123416d12ab100ec0c9e6), it explains all you need to do

Reply:
You have to remove the code below as it depends on the add you remove here. Either that or adapt the results.

Reply:
I'll replace the 10.0 nd 12.0 with 5.0 and 6.0 respectively.

Reply:
thanks!